hiro/cocoa/widget/hex-edit.cpp

```diff
@@ -5,43 +5,167 @@
 -(id) initWith:(hiro::mHexEdit&)hexEditReference {
   if(self = [super initWithFrame:NSMakeRect(0, 0, 0, 0)]) {
     hexEdit = &hexEditReference;
+    if (tableView = [[NSTableView alloc] initWithFrame:[self bounds]]) {
+      [tableView setDataSource:self];
+      // [tableView setHeaderView:nil];
+      [tableView setUsesAlternatingRowBackgroundColors:true];
+      
```

Comment:
```suggestion
      [tableView setGridStyleMask:(NSTableViewDashedHorizontalGridLineMask | NSTableViewSolidVerticalGridLineMask)];
      [tableView setRowHeight:14.0];
      
```

Makes it a bit more compact and justifies the font within the grid better: Here's a comparison:

<img width="812" alt="Screenshot 2024-12-15 at 6 30 00 PM" src="https://github.com/user-attachments/assets/caa23fa1-d573-4c0c-8e20-55913a44d455" />
<img width="875" alt="Screenshot 2024-12-15 at 6 32 02 PM" src="https://github.com/user-attachments/assets/d525e5c8-db81-418f-87e1-7b20c542e59c" />